index.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+
+// creating middleware object
+module.exports = {
+  db: [],
+  // start creates two routes to be used in development
+  start: function(app, route) {
+    // making a copy of db
+    const db = this.db;
+    return function() {
+      // creates a route to send visualization file
+      app.get(`/${route ? route : "SerVis"}`, function(req, res) {
+        res.sendFile(path.join(__dirname + '/main.html'));
+      })
+      // creates a route to send data from db
+      app.get('/ping', function(req, res) {
+        res.json(db);
+      })
+    }
+  },
+  // storing request object data into db
+  middleware: function(req, res, next) {
```

Comment:
Do we need to need req, res, next parameters here all this function does is returning a middleware?

Reply:
totally right, they were remnants from before I used closure to get access to db property, forgot to clear them out, will commit soon.